flax/experimental/nnx/examples/toy_examples/01_functional_api.py

```diff
@@ -67,10 +67,11 @@ def train_step(params, counts, batch):
   x, y = batch
 
   def loss_fn(params):
-    y_pred, (_, updates) = static.apply(params, counts)(x)
-    counts_ = updates.extract(Count)
+    model = static.merge(params, counts)
+    y_pred = model(x)
+    _counts = model.extract(Count)
```

Comment:
Why the `_` prefix? It is normally used (by convention) to indicate that a variable is unused, but here this isn't the case.

Reply:
Should we keep `counts_` then?  Here we need a new variable name, cannot reuse `counts`.

Reply:
extracted_counts or new_counts?

Reply:
`new_counts` it is